cdap-docs/reference-manual/source/java-client-api.rst

```diff
@@ -61,6 +61,38 @@ Alphabetical list:
 
 The above lists link to the examples below for each portion of the API.
 
+.. _configuring-client:
+
+Configuring your *Client
+------------------------
+
+Every *Client constructor requires a `ClientConfig` instance which configures the hostname and port of the CDAP
+instance that you wish to interact with.
+
+In a non-secure (default) CDAP instance, instantiate as follows::
+
+  // Interact with the CDAP instance located at example.com, port 10000
+  ClientConfig clientConfig = new ClientConfig("example.com", 10000);
+
+In a secure CDAP instance, first pull in the cdap-authentication-client artifact::
+
+  <dependency>
+    <groupId>co.cask.cdap</groupId>
+    <artifactId>cdap-authentication-client</artifactId>
+    <version>${cdap.client.version}</version>
+  </dependency>
+
+Then, instantiate as follows::
```

Comment:
I actually prefer "instantiate as follows: <code>" or "instantiate as shown below: <code>" rather than "instantiate as: <code>". Also "instantiate as:" is apparently incorrect grammar since a complete sentence should go before a colon.